### PR TITLE
request: cast Buffer to string

### PR DESCRIPTION
**Pull request by Raynos**
r: @kriskowal @shannili

### CODE REVIEW

**Comment by kriskowal** [comment]
:accept:

**Comment by ShanniLi** [comment]
lgtm

File: node/request.js [hunk 1]
```diff
@@ -157,7 +157,7 @@ TChannelRequest.prototype.send = function send(arg1, arg2, arg3, callback) {
         'target-service': self.serviceName,
         'service': self.headers.cn,
         // TODO should always be buffer
-        'target-endpoint': self.arg1
+        'target-endpoint': String(self.arg1)
```
Comment (jcorbin) [inline node/request.js]: well... that's an unfortunate performance hit, can we maybe then do this stat call after the critical ... resend?

**Comment by Raynos** [comment]
We should store a string arg1 on the object so we do it once instead of for every stat and trace

**Pull request merged**